components/script/stylesheet_loader.rs

```diff
@@ -244,6 +255,6 @@ impl<'a> StylesheetLoader<'a> {
 
 impl<'a> StyleStylesheetLoader for StylesheetLoader<'a> {
     fn request_stylesheet(&self, import: &Arc<RwLock<ImportRule>>) {
-        self.load(StylesheetContextSource::Import(import.clone()), "".to_owned())
+        self.load(StylesheetContextSource::Import(import.clone()), None, "".to_owned())
```

Comment:
Let's add a TODO comment about whether we should use the original loader's CORS setting. I don't find any direction from the specification about it.